serverless/packages/functions/src/model/delete_model.ts

```diff
@@ -0,0 +1,43 @@
+import { APIGatewayProxyEventV2 } from "aws-lambda";
+import { DynamoDBClient, DeleteItemCommand } from '@aws-sdk/client-dynamodb';
+
+export async function handler(event : APIGatewayProxyEventV2) {
+    let queryParams = null;
+    if (event && (queryParams = event['pathParameters']) != null) {
+        const model_id: string | undefined = queryParams['model_id'];
+        if (model_id == undefined) {
+            return {
+                statusCode: 401,
+                body: JSON.stringify({message: "Malformed request content - model ID missing."})
+            }
+        }
+        
+        const client = new DynamoDBClient({});
+
+        const command = new DeleteItemCommand({
+            TableName : "ModelTable",
+            Key :
+            {
+                model_id: {"S": model_id}
+            }
+        });
+
+        const response = await client.send(command);
+
+        if (response.$metadata.httpStatusCode == undefined || response.$metadata.httpStatusCode != 200) 
+        {
+            return {
+                statusCode: 404,
+                body: JSON.stringify({ message : "Delete operation failed" })

```

Comment:
are there any cloudwatch logs that get populated for each call of these API gateway endpoints? might be useful to be able to identify which model id we attempted to delete that resulted in the failure?

Reply:
To my knowledge, cloudwatch logs aren't populated for the API gateway endpoints outside of startup/whenever they get hit/billing info. Not sure if custom logs can be added, but interesting as something to look into.

Reply:
lets make that a followup